tox.ini

```diff
@@ -38,17 +38,14 @@ passenv =
     dev: VIA_URL
     codecov: TOXENV CI TRAVIS TRAVIS_* CODECOV_*
 deps =
-    {tests,clean,coverage}: coverage
-    {tests,lint}: httpretty
-    {tests,lint,docstrings,checkdocstrings}: pytest
-    {tests,lint,docstrings,checkdocstrings}: factory-boy
-    codecov: codecov
-    lint: prospector
-    {format,checkformatting}: black
-    {docstrings,checkdocstrings}: sphinx
-    docstrings: sphinx-autobuild
-    {tests,lint,docstrings,checkdocstrings}: -r requirements.txt
-    dev: -r requirements-dev.in
+    {clean,coverage}: -r requirements-coverage.txt
+    codecov: -r requirements-codecov.txt
```

Comment:
The separate `codecov` factor should be merged into the `tests` factor as well, so we'll have a single `tests` factor, venv and requirements file doing tests, coverage, codecov and clean.